### PR TITLE
DF-853 Sort by changes

### PR DESCRIPTION
**Pull request by JohnHeeryTNA**
Ticket URL:  https://national-archives.atlassian.net/browse/DF-853

## About these changes

Edits to the 'Sort by' dropdown element on search results pages - Desktop and mobile.

## How to check these changes

Check that the 'Sort by' dropdown reflects the changes listed in the ticket - Desktop and mobile.

## Before assigning to reviewer, please make sure you have

- [x] Checked things thoroughly before handing over to reviewer
- [x] Checked PR title starts with ticket number as per project conventions to help us keep track of changes
- [x] Ensured that PR includes only commits relevant to the ticket
- [x] Waited for all CI jobs to pass before requesting a review
- [x] Added/updated tests and documentation where relevant

## Merging PR guidance

Follow [docs\developer-guide\contributing.md](https://nationalarchives.github.io/ds-wagtail/developer-guide/contributing/)

## Deployment guidance

Follow [docs\infra\environments.md](https://nationalarchives.github.io/ds-wagtail/infra/environments/)

### CODE REVIEW

File: templates/search/blocks/search_sort_and_view_options.html [hunk 1]
```diff
@@ -5,16 +5,16 @@
 <!--    <p class="search-results__explainer">Results for everything on our website that matches your search term.</p>-->
     <form method="GET" class="search-sort-view__form search-sort-view__desktop" data-id="sort-form-desktop" id="catalogue-sort-form-desktop" data-search-type="{{ view.search_tab }}" data-search-bucket="{{ buckets.current.label }}" data-search-filter-name="Sort results" data-search-filter-value="">
 
-        <h2 class="tna-heading-l search-sort-view__heading">
-           <label for="id_sort_by_desktop">Sort by</label>
+        <h2 class="tna-heading-l search-sort-view__heading sr-only">
+           <label for="id_sort_by_desktop">Sort byx</label>
```
Comment (TNA-Allan) [inline templates/search/blocks/search_sort_and_view_options.html:9]:
```suggestion
           <label for="id_sort_by_desktop">Sort by</label>
```

**Review by TNA-Allan — CHANGES_REQUESTED** [review]
Due to typo (see suggested), tests are failing

File: templates/search/blocks/search_sort_and_view_options.html (hunk 1 above)
Comment (JohnHeeryTNA) [reply templates/search/blocks/search_sort_and_view_options.html]: Thanks @TNA-Allan, good catch I've now fixed that and tests are passing. If you could review and approve please?

File: templates/search/blocks/search_filters.html [hunk 2]
```diff
@@ -11,7 +11,7 @@ <h2 class="tna-heading-l search-sort-view__heading">
         {# render hidden inputs for fields controlled by other forms #}
         {% render_fields_as_hidden form exclude='sort_by' %}
 
-        <input type="submit" value="Update" class="search-sort-view__form-submit">
+        <input type="submit" value="Sort by" class="search-sort-view__form-submit">
```
Comment (TNA-Allan) [inline templates/search/blocks/search_filters.html:14]:
```suggestion
        <input type="submit" value="Sort" class="search-sort-view__form-submit">
```
Hi @JohnHeeryTNA just a query/suggestion if you'd like to consider - feels abrupt ending with **by** in a button

Comment (JohnHeeryTNA) [reply templates/search/blocks/search_filters.html:14]: Hmmn, that's what was suggested from the user testing, we can alway update it if there are better suggestions..

File: templates/search/blocks/search_sort_and_view_options.html [hunk 3]
```diff
@@ -14,7 +14,7 @@ <h2 class="tna-heading-l search-sort-view__heading">
 
         {% render_fields_as_hidden form exclude='sort_by' %}
 
-        <input type="submit" value="Update" class="search-sort-view__form-submit">
+        <input type="submit" value="Sort by" class="search-sort-view__form-submit">
```
Comment (TNA-Allan) [inline templates/search/blocks/search_sort_and_view_options.html:17]:
```suggestion
        <input type="submit" value="Sort" class="search-sort-view__form-submit">
```
same as previous comment.

Comment (JohnHeeryTNA) [reply templates/search/blocks/search_sort_and_view_options.html:17]: Thanks @TNA-Allan , let's go with 'Sort by' for now and I'll raise with Alex in the future if thay wish to change it.

**Review by TNA-Allan — APPROVED** [review]
Looks good. Thanks @JohnHeeryTNA

**Pull request merged**